tests/track_errors/test_ctc_errors.py

```diff
@@ -194,15 +272,257 @@ def test_ns_vertex_fn_edge():
+        for attrs in matched.gt_graph.edges.values():
+            assert attrs.get(EdgeFlag.CTC_FALSE_NEG) is True
+
+        # Pred edges with NS nodes are not penalized (see comment thread above)
+        for attrs in matched.pred_graph.edges.values():
+            assert attrs.get(EdgeFlag.CTC_FALSE_POS) is False
+
+    # CTCMatcher does not allow one gt to match multiple comp nodes.
+    # Skipping the one_to_two example
+
+    def test_correct_division(self):
+        # Check that intertrack edges have been identified
+        # No wrong semantics
+        matched = self.prep_matched(ex_graphs.good_div(1))
+        gt_edges = [(2, 4), (2, 3)]
+        pred_edges = [(6, 7), (6, 8)]
+
+        for edge, attrs in matched.gt_graph.edges.items():
+            if edge in gt_edges:
+                assert attrs.get(EdgeFlag.INTERTRACK_EDGE) is True
+            else:
+                assert attrs.get(EdgeFlag.INTERTRACK_EDGE) is False
+
+        for edge, attrs in matched.pred_graph.edges.items():
+            if edge in pred_edges:
+                assert attrs.get(EdgeFlag.INTERTRACK_EDGE) is True
+            else:
+                assert attrs.get(EdgeFlag.INTERTRACK_EDGE) is False
+            assert attrs.get(EdgeFlag.WRONG_SEMANTIC) is False
+
+    def test_fp_division(self):
+        # Check intertrack and for wrong semantic on fp division
+        matched = self.prep_matched(ex_graphs.fp_div(1))
+
+        # GT edge is not intertrack
+        attrs = matched.gt_graph.edges[(2, 4)]
+        assert attrs.get(EdgeFlag.INTERTRACK_EDGE) is False
+
+        # Pred edges are intertrack and wrong semantic
+        attrs = matched.pred_graph.edges[(6, 8)]
+        assert attrs.get(EdgeFlag.INTERTRACK_EDGE) is True
+        assert attrs.get(EdgeFlag.WRONG_SEMANTIC) is True
+
+        attrs = matched.pred_graph.edges[(6, 7)]
+        assert attrs.get(EdgeFlag.INTERTRACK_EDGE) is True
+        assert attrs.get(EdgeFlag.WRONG_SEMANTIC) is False
+
+    def test_fn_division(self):
+        # Intertrack and wrong semantic check
+        matched = self.prep_matched(ex_graphs.one_child(1))
+
+        # Pred edge
+        attrs = matched.pred_graph.edges[(6, 8)]
+        assert attrs.get(EdgeFlag.INTERTRACK_EDGE) is False
+        assert attrs.get(EdgeFlag.WRONG_SEMANTIC) is True
+
+        # Gt edges are just intertrack
+        attrs = matched.gt_graph.edges[(2, 4)]
+        assert attrs.get(EdgeFlag.INTERTRACK_EDGE) is True
+        attrs = matched.gt_graph.edges[(2, 3)]
+        assert attrs.get(EdgeFlag.INTERTRACK_EDGE) is True
+
+    def test_wrong_child(self):
+        # Intertrack and wrong semantic check
+        matched = self.prep_matched(ex_graphs.wrong_child(1))
+
+        # One pred edge correct
+        attrs = matched.pred_graph.edges[(7, 8)]
+        assert attrs.get(EdgeFlag.INTERTRACK_EDGE) is True
+        assert attrs.get(EdgeFlag.WRONG_SEMANTIC) is False
+        # Not wrong semantic b/c edge doesn't directly match to gt edge
```

Comment:
So should this edge also be FP in that case? As both nodes are present in GT but the edge is not?